packages/components/src/Combobox/Combobox.test.tsx

```diff
@@ -437,6 +437,7 @@ function renderCombobox() {
       multiSelect={mockMultiSelectValue()}
       selected={mockSelectedValue()}
       onSelect={handleSelect}
+      debounce={0}
```

Comment:
debatable if this is the best way to handle this or not

it does make tests happy and avoid the need to do any kind of timer manipulation because otherwise tests will make their assertions too soon

Reply:
can use fake timers (`jest.useFakeTimers()`) and `await jest.runAllTimersAsync();` to advance it past the debounce